lsp-bridge.el

```diff
@@ -933,6 +940,17 @@ So we build this macro to restore postion after code format."
   (not (or (member lsp-bridge-last-change-command lsp-bridge-completion-stop-commands)
            (member (format "%s" last-command) lsp-bridge-completion-stop-commands))))
 
+(defun lsp-bridge-string-interpolation-p (string-interpolation-open-chars-alist)
+  "Check if the cursor position is subject to string interpolation"
+  (save-excursion
+    (let* ((initial-pos (point))
+           (search-char (cdr (or (assoc (buffer-local-value 'major-mode (current-buffer))
+                                        string-interpolation-open-chars-alist)
+                                 (assoc t string-interpolation-open-chars-alist))))
+           (open-pos (search-backward-regexp search-char nil t)))
```

Comment:
You need use `save-excursion` wrap `search-backward-regexp` outside, then you can remove initial-pos, it's unnecessary.

Reply:
Fixed in [151cb8f](https://github.com/manateelazycat/lsp-bridge/pull/491/commits/151cb8fcfb09b583719eb3bc5e0b3d8f09d87d99)